src/test/java/StepDefinitions/BaseTest.java

```diff
@@ -16,7 +16,6 @@ public class BaseTest {
     @BeforeClass
     public void setUp() {
         System.setProperty("webdriver.chrome.driver", "C:\\Users\\rudi\\IdeaProjects\\CucumberTestNG\\src\\main\\resources\\drivers\\chromedriver.exe");
-//        System.setProperty("webdriver.http.factory", "jdk-http-client");
```

Comment:
It's not needed to create BaseTest class because setUp() and tearDown() methods should be placed to Steps class as steps